src/test/java/mate/academy/spring/boot/controllerTests/ShoppingCartControllerTest.java

```diff
@@ -0,0 +1,309 @@
+package mate.academy.spring.boot.controllerTests;
```

Comment:
```suggestion
package mate.academy.spring.boot.controller;
```
use the same package names (fix it everywhere)